Dockerfile

```diff
@@ -1,6 +1,7 @@
 ARG MENDER_CLI_VERSION=1.11.0
 ARG MENDER_ARTIFACT_VERSION=3.10.1
 ARG MENDER_CLIENT_VERSION=3.5.1
+ARG MENDER_APP_UPDATE_MODULE_VERSION=master
```

Comment:
We should set to 1.0.0 once it is released.

I personally would wait for this task to be done after the tagging - but I was not part of the planning and didn't provide such input, so it is okey too to merge as-is and have a follow-up 0SP task for this.